lua/catppuccin/lib/vim/compiler.lua

```diff
@@ -59,11 +59,18 @@ let g:colors_name = "catppuccin"]],
 	end
 	table.insert(lines, "]]end)")
 	if vim.fn.isdirectory(O.compile_path) == 0 then vim.fn.mkdir(O.compile_path, "p") end
-	local file = io.open(O.compile_path .. path_sep .. flavour .. "_compiled.lua", "wb")
+	local file = io.open(O.compile_path .. path_sep .. flavour, "wb")
 	local ls = load or loadstring
+
+	if vim.g.catppuccin_debug then -- Debugging purpose
+		local deb = io.open((path_sep == "/" and "/tmp" or os.getenv "TMP") .. "/catppuccin_debug.lua", "wb")
```

Comment:
This is more of a question, when compile is called it is called in the context of looping though all flavours. This file will be overridden by the next compile call. Should the path be `/tmp/catppuccin/name.lua`?